### PR TITLE
async await wonkiness and shuffle the notes

### PR DESCRIPTION
**Pull request by taddison92**

### CODE REVIEW

File: frontend/src/components/App.tsx [hunk 1]
```diff
@@ -22,12 +22,30 @@ export default function App() {
...
+
+      // And swap it with the current element.
+      [notes[currentIndex], notes[randomIndex]] = [
+        notes[randomIndex], notes[currentIndex]];
+    }
+
+    return notes;
+  }
+
+  const handleClickGenerate = async (): Promise<void> => {
```
Comment (taddison92) [inline frontend/src/components/App.tsx:44]: the main change is making this function async

File: frontend/src/components/App.tsx [hunk 2]
```diff
@@ -40,9 +58,11 @@ export default function App() {
       {name: 'e', duration: '4|', timeBetweenNotes: waitTimeInMilliseconds * 2},
     ]
 
+    shuffle(notes);
+
     let i = 0;
     do {
-      renderNote(notes[i]);
+      await renderNote(notes[i]);
```
Comment (taddison92) [inline frontend/src/components/App.tsx:65]: and awaiting the renderNote call

File: frontend/src/components/App.tsx [hunk 3]
```diff
@@ -40,9 +58,11 @@ export default function App() {
       {name: 'e', duration: '4|', timeBetweenNotes: waitTimeInMilliseconds * 2},
     ]
 
+    shuffle(notes);
```
Comment (taddison92) [inline frontend/src/components/App.tsx:61]: this guy is just to randomize the notes

**Comment by ahite89** [comment]
Thanks bud!

**Pull request merged**